src/documents/IoCReferences.md

```diff
@@ -98,7 +103,7 @@ IoC references may be used almost anywhere within a component's options, for exa
 For a conventional IoC reference (of the style `<componentRef>` rather than the style `<iocss expression>`), a search is begun upwards from the site of the reference in the component tree to find the first component which matches the context name. 
 The following diagram shows a possible such reference site in green:
 
-**NOTE**: The following diagram needs to be updated to reflect the Infusion 2.0 visibility rules
+<div class="infusion-docs-note"><strong>Note:</strong> The following diagram needs to be updated to reflect the Infusion 2.0 visibility rules - the "static environment" and "dynamic environment" no longer exist.</div>
```

Comment:
Hi @jhung - thanks for the improvements.
i) No numbers should be duplicates - so the numbers which are "1" at the left should increment in the usual way.
ii) All of the things that you have labelled with numbers 8 through 11 are NOT resolvable via resolveRoot. Only the thing labelled x / 7 is.
iii) the thing labelled / should be at the head of the WHOLE tree - that is, it should replace the thing labelled 5. There should just be a single global tree in the whole diagram
iv) Therefore the thing labelled "7" should be a kind of shadow which becomes the sibling of 3 and 6
v) The global root itself is not currently resolvable